internal/controller/controllers/image_controller.go

```diff
@@ -67,6 +67,35 @@ func (r *ImageReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 }
 
 func (r *ImageReconciler) create(ctx context.Context, image *adiiov1alpha1.Image) (ctrl.Result, error) {
+	covertStaticIPConfig := func(crdStaticIPConfig []*adiiov1alpha1.StaticIPConfig) []*models.StaticIPConfig {
```

Comment:
this function is already long enough, should probably be outside as a helper function.

Reply:
Done